docs/topics/downloader-middleware.rst

```diff
@@ -1020,10 +1020,10 @@ the request will be ignored by this middleware even if
 RobotFileParser
 ~~~~~~~~~~~~~~~
 
-`RobotFileParser <https://docs.python.org/3.7/library/urllib.robotparser.html>`_ is 
-Python's inbuilt robots.txt_ parser. The parser is fully compliant with `Martijn Koster's 
+`RobotFileParser <https://docs.python.org/3.7/library/urllib.robotparser.html>`_ is
+Python's built-in robots.txt_ parser. The parser is fully compliant with `Martijn Koster's
 1996 draft specification <http://www.robotstxt.org/norobots-rfc.txt>`_. It lacks
-support for wildcard matching. Scrapy uses this parser by default.
+support for wildcard matching.
```

Comment:
Probably makes sense to add a note on why this parser is useful:
* faster than protego
* compatibility with previous Scrapy versions

Reply:
RobotFileParser is still mentioned as default in docs above, where all parsers are listed. Could you please re-read the whole docs about robots.txt, and check that they make sense, assuming protego is a default parser?